client/src/pages/Home.tsx

```diff
@@ -105,7 +108,8 @@ function Home(this: any) {
   // Testing function to call Summary analysis fn...
   const getSummary = async () => {
     setSubmitted(true);
-    console.log("submitted: " + submitted);
+    setSummaryLoading(true);
+    setSummaryError(false);
     const body = JSON.stringify({ text: inputValue , summary_len_option: "default"});
     await fetch("/api/summary/process", { ...postRequestOptions, body })
```

Comment:
potentially could make use of `try` block when using `catch` and `finally`

Reply:
I think this is fine, JS has a built in thing with all async functions that omits the `try` block (i.e. await(x).then(y).catch(z).finally(a)` works fine

```
await someAsyncFunction()
  .then(result => {
    // Handle the result
    return anotherAsyncFunction(result);
  })
  .then(anotherResult => {
    // Handle anotherResult
    return yetAnotherAsyncFunction(anotherResult);
  })
  .catch(error => {
    // Handle errors from any of the above Promises
  })
  .finally(() => {
    // Code to run regardless of success or failure
  });
```

Reply:
`catch` and `finally` in this case is especailyl cool, because it can catch any errors in any of the promises in the `then` chain